src/plays/self-clicking-button/SelfClickingButton.js

```diff
@@ -0,0 +1,54 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import { useState, useEffect, useRef } from 'react';
+
+// WARNING: Do not change the entry componenet name
+function SelfClickingButton(props) {
+  // Your Code Start below.
+  const buttonRef = useRef(null);
+  const [counter, setCounter] = useState(0);
+
+  function clickHandler(event) {
+    setCounter((prev) => prev + 1);
+  }
+
+  useEffect(() => {
+    buttonRef.current.addEventListener('click', clickHandler);
+
+    const interval = setInterval(() => {
+      buttonRef.current.click();
+    }, 2000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, []);
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
+          <div className="self-click">
+            <button ref={buttonRef}> Self Click Button</button>
+            <p className="output">
+              At{' '}
+              <u>
+                {new Date().getHours() +
```

Comment:
you might want to format it in ##:##:## template so that content will not jump every time time is updated

Reply:
Do you mean padding with a 0 or to format it?

Reply:
Taken care